### DISCUSSION TITLE
can we choose enable/disable NCCL after install horovod with NCCL?

### DISCUSSION THREAD

**Discussion by hyxcl** [horovod/horovod#3567] [General]
I searched horovod with NCCL, and most materials are about install horovod with NCCL.
after we installed horovod with NCCL, is there any setting for us to choose whether to use NCCL?   if we don't want to use NCCL in some case, we must install another horovod without NCCL?

**Comment by maxhgerlach** [comment]
I don't think there is a runtime toggle to disable NCCL in Horovod. 

Would it help in your case to place certain communication ops onto CPU? In that case those wouldn't use NCCL. Of course, I'd  imagine that this would also preclude MPI-aware GPU in case that's a concern to you.
